app/views/auth_source_ldaps/_form.html.erb

```diff
@@ -1,7 +1,5 @@
 <% javascript 'auth_source_ldap' %>
 
-<% title_actions link_to(icon_text("question-sign", _("Documentation"), :class => "icon-white"), "http://www.theforeman.org/manuals/#{SETTINGS[:version].short}/index.html#4.1.2LDAPAuthentication", :rel => "external", :class => "btn btn-info") %>
```

Comment:
This docs button was simply moved from the form to the index page to make it more consistent with the rest. Besides, it looked really weird as it was creating a title actions bar for just that button and moving the content of the form (inside the square) some pixels below it.